src/components/accordion/accordion.style.ts

```diff
@@ -1,56 +1,78 @@
+/* eslint-disable no-nested-ternary */
 import styled, { css } from "styled-components";
+import { space, margin, MarginProps } from "styled-system";
 
-import { space, margin } from "styled-system";
-
+import Box from "../box";
 import Icon from "../icon";
-import { baseTheme } from "../../style/themes";
-import ValidationIconStyle from "../../__internal__/validations/validation-icon.style";
 import addFocusStyling from "../../style/utils/add-focus-styling";
+import { baseTheme } from "../../style/themes";
 
-const StyledAccordionGroup = styled.div`
-  ${margin}
-`;
-
-export interface StyledAccordionContainerProps {
-  /** Toggles left and right borders, set to none when variant is subtle */
+interface StyledWrapperProps extends MarginProps {
   borders?: "default" | "full" | "none";
-  /** Sets accordion width */
+  size?: "large" | "small";
+  width?: string;
+  variant?: "standard" | "subtle";
+}
+interface StyledDetailsProps {
+  disableContentPadding?: boolean;
+  iconAlign?: "left" | "right";
+  reduceMotion?: boolean;
+  size?: "large" | "small";
+  variant?: "standard" | "subtle";
+}
+interface StyledSummaryTitleProps {
+  size?: "large" | "small";
+  variant?: "standard" | "subtle";
+}
+interface StyledContentProps {
+  expanded?: boolean;
+  reduceMotion?: boolean;
+  variant?: "standard" | "subtle";
   width?: string;
-  /** Sets accordion variant */
+}
+interface StyledIconProps {
   variant?: "standard" | "subtle";
 }
 
-const StyledAccordionContainer = styled.div<StyledAccordionContainerProps>`
-  ${space}
-  display: flex;
-  ${({ variant }) =>
-    variant &&
-    css`
-      align-items: stretch;
-    `};
-  justify-content: center;
-  flex-direction: column;
-  box-sizing: border-box;
+const defaultMaxHeightTransitionMs = "100ms";
```

Comment:
suggestion (non-blocking): it might be worth using capitalised text here so we're consistent with how we've named constant values elsewhere in the code base

```
const DEFAULT_MAX_HEIGHT_TRANSITION_MS  = "100ms";